resources/assets/components/Campaign/index.js

```diff
@@ -133,7 +133,7 @@ const Campaign = ({ id }) => {
         <p>{format(parse(campaign.startDate), 'MM/D/YYYY')}</p>
         <h4>End Date</h4>
         <p>
-          {campaign.end_date
+          {campaign.endDate
```

Comment:
Oh snap, so was this always showing as `-` even if there really was an end date?

Reply:
Apparently!